netci.groovy

```diff
@@ -308,9 +308,10 @@ def targetGroupOsMap = ['netcoreapp': ['Windows 10', 'Windows 7', 'Windows_NT',
 
                 def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
                     // On Windows we use the packer to put together everything. On *nix we use tar
+                    def skipTests = targetGroup == 'netfx' ? '' : '-SkipTests';
                     steps {
                         batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd -${configurationGroup} -framework:${targetGroup}")
-                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build-tests.cmd -${configurationGroup} -framework:${targetGroup} -SkipTests")
```

Comment:
why change here? This is for building, netfx isn't special.
Isn't the right change to line 370 was
```
[true, false].each { isPR ->
    ['netcoreapp'].each { targetGroup ->
        ['Debug', 'Release'].each { configurationGroup ->
```
to
```
[true, false].each { isPR ->
    ['netcoreapp', 'netfx'].each { targetGroup ->
        (targetGroupOsMap[targetGroup]).each { osName ->   
```